app/src/components/DraftResourceButtons.tsx

```diff
@@ -0,0 +1,46 @@
+import { Text, Paper, Stack, Center, Loader } from '@mantine/core';
+import { trpc } from '@/util/trpc';
+import SidebarButtonList from './SidebarButtonList';
+
+/**
+ * Component which retrieves all resources and their counts and translates them into buttons and adds
+ * a search button that links to the overall search page
```

Comment:
Do we want to add a search button here or just remove this part of the component description?

Reply:
I would also specify that this is for draft resources